packages/core/contracts/financial-templates/expiring-multiparty/PricelessPositionManager.sol

```diff
@@ -907,7 +933,7 @@ contract PricelessPositionManager is Lockable {
         }
     }
 
-    function _transformPriceIdentifier(uint256 requestTime) internal view returns (bytes32) {
+    function _transformPriceIdentifier(uint256 requestTime) public view returns (bytes32) {
```

Comment:
can we revert this change to match the release code?

Reply:
this is happening while committing to git. Theere is git hook configured that checks for indentation and then commits.

Reply:
oh, but the hook wouldn't change internal to public? There is a way to skip the hook while committing

Reply:
yeah

Reply:
@ShubhamChndrvnshi I still see this as public. Can we make this internal?